tests/integration/helpers.py

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Charm integration test helpers."""
+
+import logging
+from pathlib import Path
+
+import requests
+import yaml
+from pytest_operator.plugin import OpsTest
+from temporal_client.activities import say_hello
+from temporal_client.workflows import SayHello
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./charmcraft.yaml").read_text())
+APP_NAME_AIRBYTE_SERVER = METADATA["name"]
+APP_NAME_TEMPORAL_SERVER = "temporal-k8s"
+APP_NAME_TEMPORAL_ADMIN = "temporal-admin-k8s"
+APP_NAME_TEMPORAL_UI = "temporal-ui-k8s"
+
+GET_HEADERS = {"accept": "application/json"}
+POST_HEADERS = {"accept": "application/json", "content-type": "application/json"}
+
+
+def get_airbyte_charm_resources():
+    return {
+        "airbyte-api-server": METADATA["resources"]["airbyte-api-server"]["upstream-source"],
+        "airbyte-bootloader": METADATA["resources"]["airbyte-bootloader"]["upstream-source"],
+        "airbyte-connector-builder-server": METADATA["resources"]["airbyte-connector-builder-server"][
+            "upstream-source"
+        ],
+        "airbyte-cron": METADATA["resources"]["airbyte-cron"]["upstream-source"],
+        "airbyte-pod-sweeper": METADATA["resources"]["airbyte-pod-sweeper"]["upstream-source"],
+        "airbyte-server": METADATA["resources"]["airbyte-server"]["upstream-source"],
+        "airbyte-workers": METADATA["resources"]["airbyte-workers"]["upstream-source"],
+    }
+
+
+async def run_sample_workflow(ops_test: OpsTest):
```

Comment:
what is this for?

Reply:
I realized it wasn't being called so I updated it, but this is to run a sample workflow on Temporal to ensure it's up and running.